### DISCUSSION TITLE
Identify speaker?

### DISCUSSION THREAD

**Discussion by pinballelectronica** [linto-ai/whisper-timestamped#99] [Ideas]
Amazon has a mechanism in their Polly tool to identify speakers-  it's not great but it sort of works.   Is this something attainable by whisper (timestamped)?  It seems like the Timestamped version would have to be the fork if even possible.

**Comment by BadarSaghir** [comment]
may be this would help
https://github.com/MahmoudAshraf97/whisper-diarization

**Comment by dchapelet** [comment]
Hi,
Have you planned in your roadmap _**whisper_timestamped**_ to be able to detect speakers in your Automatic Speech Recognition?
Or or I have to use _**whisper-diarization**_ ?
Thank you very much for your answer.
Kind regards,
David
